libs/s25main/mapGenerator/RandomMap.cpp

```diff
@@ -251,16 +244,19 @@ namespace rttr { namespace mapGenerator {
         const auto waterNodes = std::count(map_.z.begin(), map_.z.end(), map_.height.minimum);
```

Comment:
Maybe cast this here to unsigned to avoid the warning below:

```
/source/libs/s25main/mapGenerator/RandomMap.cpp: In member function 'void rttr::mapGenerator::RandomMap::CreateWaterMap()':
/source/libs/s25main/mapGenerator/RandomMap.cpp:257:23: error: comparison between signed and unsigned integer expressions [-Werror=sign-compare]
         if(waterNodes > map_.players * islandSize)
```